resources/aries-test-b/droplets.tf

```diff
@@ -7,6 +7,21 @@ resource "digitalocean_droplet" "aries-test-b-nodes-farmer-relayer" {
     data.digitalocean_ssh_key.nazar-key.id,
     data.digitalocean_ssh_key.serge-key.id,
     data.digitalocean_ssh_key.leo-key.id,
+    # TODO: Add floating IP so that you can add this without changing the IP address
+    # data.digitalocean_ssh_key.sam-key.id,
+  ]
+}
+
+resource "digitalocean_droplet" "aries-relaynet-a" {
```

Comment:
Why do we have `relaynet-a` in `test-b` file?

Reply:
This was an error when it was deployed , and to change this now will bring down the current network , as the server would be redeployed